packages/foundation/src/responsive/viewport.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
 
 import { Breakpoints } from '../types';
-import { useWindowDimensions } from '../hooks/use-window-dimensions';
-import { BreakpointsSizes } from './breakpoints-sizes';
+import { useViewport } from '../hooks';
 
-type BreakpointString = 's|m' | 'm|l' | 'l|xl';
+type BreakpointString = 's|m' | 'm|l' | 'l|xl' | 'm|l|xl';
```

Comment:
Adding `m|l|xl` combination, so the viewport renders the content on all breakpoints but small